stack/Pulumi.seqr.yaml

```diff
@@ -0,0 +1,11 @@
+encryptionsalt: v1:58BWphXSDio=:v1:9KydLgEdNGdRloZf:phZCGaeRzAGsTdOdTWBG1Wbu6GtX3w==
+config:
+  datasets:archive_age: "365"
```

Comment:
Not sure we'll actually use the `archive` bucket for seqr, but maybe 30 days is enough?